mindsdb/__init__.py

```diff
@@ -2,11 +2,12 @@
 import sys
 import json
 
-from mindsdb.__about__ import __version__
-from mindsdb.__about__ import __version__ as mindsdb_version
-from mindsdb.utilities.fs import get_or_create_data_dir, create_dirs_recursive
-from mindsdb.utilities.functions import args_parse, is_notebook
-from mindsdb.utilities.telemetry import telemetry_file_exists, disable_telemetry
+from .__about__ import __version__
+from .__about__ import __version__ as mindsdb_version
+from .utilities.fs import get_or_create_data_dir, create_dirs_recursive
+from .utilities.functions import args_parse, is_notebook
+from .utilities.telemetry import telemetry_file_exists, disable_telemetry
```

Comment:
why imports are relative?

Reply:
It is quite strange but with absolute imports any `flask db` command returns import error.